stock_move_view_adj/views/stock_move_line_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_move_line_tree" model="ir.ui.view">
+        <field name="name">stock.move.line.tree</field>
+        <field name="model">stock.move.line</field>
+        <field name="inherit_id" ref="stock.view_move_line_tree" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='reference']" position="after">
```

Comment:
```suggestion
            <xpath expr="//field[@name='location_dest_id']" position="after">
```

This is my personal preference, but the customer may think otherwise.